apps/fluent-tester/src/TestComponents/Link/LinkTest.tsx

```diff
@@ -15,7 +15,7 @@ const Links: React.FunctionComponent = () => {
   return (
     <Stack style={stackStyle}>
       <Link url="https://www.bing.com/" content="Click to navigate." />
-      <Link onPress={doPress} content="Click to alert." />
+      <Link disabled onPress={doPress} content="Click to alert." />
```

Comment:
Hmm maybe do a separate entry/test for the disabled case? It feels like having a proper test case for onPress would be useful separate from testing that disabled doesn't fire the onPress callback.

Reply:
oh this isn't testing the new component. I think I was trying web debugging a scenario. I'll delete this bit. All the testing for the new Link should be under Experimental Link